conf/application.conf

```diff
@@ -146,4 +146,8 @@ microservice {
   }
 }
 
-feature.flag.memorandum.enabled = "true"
\ No newline at end of file
+feature.flag.memorandum.enabled = "true"
+
+circuit.breaker.failedMemorandumCallsInUnstableBeforeUnavailable = 60
```

Comment:
nesting the names in objects would read better. Something like:

```
circuit.breaker {
  memorandum {
    failedCallsinUnstableBeforeUnavailable = 60
    unavailablePeriodDurationInMillis = 60000 # 1 minute
    unstablePeriodDurationInMillis = 60000 # 1 minute
  }

  if {
    failedCallsinUnstableBeforeUnavailable = 60
    unavailablePeriodDurationInMillis = 60000 # 1 minute
    unstablePeriodDurationInMillis = 60000 # 1 minute
  }
}

```

You will need to update `app-conf-*` files as well